L4 Homework Glaschenko A

glaschenko opened the pull request:
Не особо понял зачем при наличии класса модели еще и класс DTO и куча бойлерплейта чтобы мапить одно на другое. Замапил json сразу на класс модели. Какие у этого минусы?

glaschenko committed:
Не особо понял зачем при наличии класса модели еще и класс DTO и куча бойлерплейта чтобы мапить одно на другое. Замапил json сразу на класс модели. Какие у этого минусы?

glaschenko committed:
Merge branch 'master' of https://github.com/guid-empty/otus-cocktail-app-lessons

glaschenko committed:
search box

glaschenko committed:
categories list

guid-empty commented:
ответил в чате по домашним работам
к сожалению, эту работу принять пока не могу

дальше предлагаю обсуждение перенести в чат по домашним работам

glaschenko committed:
complete task

glaschenko committed:
merge with master

erge branch 'master' of https://github.com/guid-empty/otus-cocktail-app-lessons

glaschenko commented:
fixes required
